Bootstrap/BootstrapTests/ACustomFolder/BootstrapCustomFolderTests.swift

```diff
@@ -12,19 +12,31 @@ final class BootstrapCustomFormatTests: QuickSpec {
             beforeEach {
                 setNimbleTestFolder("CustomFolder")
                 view = UIView(frame: CGRect(origin: .zero, size: CGSize(width: 44, height: 44)))
-                view.backgroundColor = .blue
+                view.backgroundColor = UIColor { traits -> UIColor in
+                    if traits.userInterfaceStyle == .dark {
+                        return .brown
+                    } else {
+                        return .blue
+                    }
+                }
             }
 
             it("fails to find the snapshots due to the custom folder") {
                 expect(view).notTo(haveValidSnapshot(named: "something custom"))
+                expect(view).notTo(haveValidSnapshot(named: "something custom"))
             }
 
             it("finds the snapshots using a custom images directory") {
-                expect(view).to(haveValidSnapshot())
+                expect(view).to(recordSnapshot())
             }
 
             it("finds device agnostic snapshots with custom images directory") {
-                expect(view).to(haveValidDeviceAgnosticSnapshot())
+                expect(view).to(recordDeviceAgnosticSnapshot())
+            }
+
+            it("find the snapshot using a custom image directory for light and dark mode") {
+                expect(view).to(recordSnapshot(userInterfaceStyle: .light))
+                expect(view).to(recordSnapshot(userInterfaceStyle: .dark))
```

Comment:
I guess all of those also should be `haveValid` and not `record`

Reply:
I can't get this test to run properly, we still need to record images for this test, that is why I left it like that.

Reply:
Here we should stick to  AAA (A.K.A Arrange, Act, and Assert) principles, and as we expect two different outputs , should be two isolated unit tests